GameData/ROEngines/PartConfigs/RL10A4_SSTU.cfg

```diff
@@ -137,19 +137,7 @@ PART
 	@MODULE[ModuleEngineConfigs]
 	{
 		@configuration = RL10A-4
-		!CONFIG[RL10A-1] {}
-		!CONFIG[RL10A-3-1] {}
-		!CONFIG[RL10A-3-3] {}
-		!CONFIG[RL10A-3-3A] {}
-		!CONFIG[RL10A-4-2N] {}
-		!CONFIG[RL10A-5] {}
-		!CONFIG[RL10B-2] {}
-		!CONFIG[RL10C-1] {}
-		!CONFIG[RL10C-1-1] {}
-		!CONFIG[RL10C-2-1] {}
-		!CONFIG[RL10C-3] {}
-		!CONFIG[CECE-Base] {}
-		!CONFIG[CECE-High] {}
-		!CONFIG[CECE-Methane] {}
+		!CONFIG:HAS[~name[RL10A-4*]] {}
+		!CONFIG[RL10A-4-2*] {}
```

Comment:
Here's a neat thing about MM:  `HAS` implies `,*`.  So `!CONFIG:HAS[~name[RL10A-4*]] {}` will delete everything that does not have the RL10A-4* name.  However, `!CONFIG[RL10A-4-2*] {}` will only delete the _first_ CONFIG that matches name = RL10A-4-2*.

I guess you want `!CONFIG:HAS[~name[RL10A-4]] {}` here, you just want A-4 with no extensions to survive?  Not certain what the intended result is.

Reply:
i want it to keep any A4 config except the A4-2 & A4-2N

Reply:
`!CONFIG[RL10A-4-2*],* {}` should do that.  (Delete all configs whose name matches RL10A-4-2*)